manager/controllers/app/m4dapplication_controller.go

```diff
@@ -23,36 +27,44 @@ import (
 	pc "github.com/ibm/the-mesh-for-data/pkg/policy-compiler/policy-compiler"
 	"google.golang.org/grpc/codes"
 	"google.golang.org/grpc/status"
-	corev1 "k8s.io/api/core/v1"
-	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+const (
+	// OwnerLabelKey is a key to Labels map.
+	// All owned resources should be labeled using this key.
+	OwnerLabelKey string = "m4d.ibm.com.owner"
```

Comment:
I know that it was like that before, but I think usually `/` is used after the domain rather than a dot.

Reply:
Updated.